### PR TITLE
Pass resources loading state to Create VM Wizard

### PR DESCRIPTION
**Pull request by rawagner**
https://github.com/kubevirt/web-ui-components/pull/99

### CODE REVIEW

File: frontend/public/kubevirt/components/vm.jsx [hunk 1]
```diff
@@ -314,27 +314,47 @@ const mapDispatchToProps = () => ({
...
+const ConnectedNewVMWizard = ({ activeNamespace, resources, onHide, createVm }) => {
+  const namespaces = getLoadingResource(resources, NamespaceModel.kind);
+  const templates = getLoadingResource(resources, TemplateModel.kind);
+  const networkConfigs = getLoadingResource(resources, NetworkAttachmentDefinitionModel.kind);
+  const storageClasses = getLoadingResource(resources, StorageClassModel.kind);
+  const persistentVolumeClaims = getLoadingResource(resources, PersistentVolumeClaimModel.kind);
+
+  const error = Object.keys(resources).find(key => resources[key].loadError);
+  if (error) {
+    errorModal({error});
```
Comment (suomiy) [inline frontend/public/kubevirt/components/vm.jsx]: I started using this as well
little enhancement: 

```jsx
const Errors = ({errors}) => (
  <ul>
    {errors.map((error, idx) => <li key={idx}>{error}</li>)}
  </ul>
);
errorModal({
  error: (<Errors errors={errors} />)
});
```

Comment (rawagner) [reply frontend/public/kubevirt/components/vm.jsx]: @suomiy will you create a PR with just `Error` component ?

**Comment by suomiy** [comment]
Implented in #111 with more features

**Comment by rawagner** [comment]
https://github.com/kubevirt/web-ui/pull/111 merged, closing this one

**Pull request closed without merging**